### PR TITLE
Bump python-pulp-deb to 2.6.1

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/python-pulp-deb/python-pulp-deb.spec [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
-Version:        2.4.0
-Release:        0.1.b1%{?dist}
+Version:        2.6.1
+Release:        1%{?dist}
 Summary:        pulp-deb plugin for the Pulp Project
 
 License:        GPLv2+
-URL:            https://pulpproject.org/#deb
-Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}b1.tar.gz
+URL:            https://pulpproject.org
```
Comment (evgeni) [inline packages/python-pulp-deb/python-pulp-deb.spec:10]: @quba42 if you want to keep that `#deb` anchor you had before, please set the anchored URL in setup.py, then the script will automatically update it :)

Comment (quba42) [reply packages/python-pulp-deb/python-pulp-deb.spec:10]: From opening it just now, it does not look like the `#deb` anchor still leads anywhere useful, so I guess it is fine to drop it?

**Review by quba42 — APPROVED** [review]
Looks sane to me, but I am no packaging expert :wink:.

File: packages/python-pulp-deb/python-pulp-deb.spec (hunk 1 above)
Comment (evgeni) [reply packages/python-pulp-deb/python-pulp-deb.spec:10]: Probably, yeah.

What I really wanted to say: if you ever have a more useful page than the main homepage of pulp, just drop it into setup.py and the rest will happen automatically :)

Comment (quba42) [reply packages/python-pulp-deb/python-pulp-deb.spec:10]: I will try to keep it in mind. :wink:

File: packages/python-pulp-deb/python-pulp-deb.spec [hunk 2]
```diff
@@ -1,36 +1,38 @@
...
+Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
```
Comment (ekohl) [inline packages/python-pulp-deb/python-pulp-deb.spec:11]: @evgeni Any idea why pyp2rpm doesn't use `%{pypi_source}`? It'd be much nicer.

**Review by ekohl — APPROVED** [review]

File: packages/python-pulp-deb/python-pulp-deb.spec (hunk 2 above)
Comment (evgeni) [reply packages/python-pulp-deb/python-pulp-deb.spec:11]: I think I disabled that in our template as it didn't work for the modules we rename. Or something like that. we can revisit that

**Pull request merged**